src/Docker/Runner/StateFile.php

```diff
@@ -16,6 +16,8 @@
 
 class StateFile
 {
+    const NAMESPACE_PREFIX = 'component';
```

Comment:
bylo by teda dobry overit, ze se v soucasnych statech tenhle klic nevyskytuje

Reply:
✅ je to jen uložených popiskách dat v genericu